frontend/components/SettingsPage/index.tsx

```diff
@@ -16,31 +17,18 @@ import { Optional } from '@/types/Util';
 
 import { AddressLink } from '../AddressLink';
 import { CustomAlert } from '../Alert';
-import { CardTitle } from '../Card/CardTitle';
-import { GoToLoginPageButton } from '../Pages/GoToLoginPageButton';
-import { GoToMainPageButton } from '../Pages/GoToMainPageButton';
 import { CardSection } from '../ui/CardSection';
-import { DebugInfoSection } from './DebugInfoSection';
 
 const { Text, Paragraph } = Typography;
 
-const SettingsTitle = () => (
-  <CardTitle
-    title={
-      <Flex gap={10}>
-        <SettingOutlined />
-        Settings
-      </Flex>
-    }
-  />
-);
-
 const NoBackupWallet = () => {
   const { goto } = usePageState();
 
   return (
     <>
-      <Text type="secondary">No backup wallet added.</Text>
+      <Text type="secondary" style={{ marginLeft: 52 }}>
```

Comment:
why do we need such a big marginLeft? Isn't it supposed to be aligned with the title?
<img width="386" height="102" alt="image" src="https://github.com/user-attachments/assets/64b58513-aed1-44fc-968b-96a203029021" />